.github/ISSUE_TEMPLATE/default-new.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Issue
+about: This template is for a generic new issue.
+---
+
+
+<!-- This template provides suggestions for what you might want to add to your issue. Feel free to add or exclude sections.-->
+
+**Summary**
+
+<!-- Tell us a bit about this issue. Is there a project you'd like to work on, a feature request, a bug, or do you have a question for the team? -->
+
+**Acceptance Criteria**
+
+<!-- When can we resolve this issue? What needs to be fixed, answered, or built before this is considered "done"? -->
+
+**Do you want to work on this issue (Yes/No)?**
+
+<!-- Let us know if you're interested in working on this issue. -->
```

Comment:
*Suggestion:*

Add something like:

1. `<!--Tag any relevant parties for review for this issue. -->`
2. Include a `cc / @pyladies/communications ` to tag and notify the entire comms team at the bottom here.

Reply:
I'll add that section